devstack_vm/bin/run_tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
this is not the correct source. The base should be: https://github.com/cloudbase/ci-overcloud-init-scripts/blob/test_new_tempestrun/scripts/devstack_vm/bin/run-all-tests.sh